pytorch_lightning/loops/utilities.py

```diff
@@ -62,6 +64,20 @@ def _extract_hiddens(training_step_output: STEP_OUTPUT, truncated_bptt_steps: in
     return hiddens
 
 
+# rank_zero_warning of type UserWarning for max_epochs if not set
+def _parse_max_epochs_and_steps(
+    max_steps: int, max_epochs: int, max_time: Union[str, timedelta, Dict[str, int]], min_steps: int, min_epochs: int
+) -> Tuple[int, int]:
+    if max_epochs is None and max_steps == -1 and max_time is None:
+        rank_zero_warn(
+            "`max_epochs` is not set. By default it will run for 1000 epochs.",
```

Comment:
```suggestion
            "`max_epochs` is not set. By default it will run for 1000 epochs.",
```
The message is not accurate. 
1) "it" here refers to "max_epochs" from the previous sentence, but this is not what you meant to express.
2) The loop is not guaranteed to run for 1000 epochs.

```suggestion
            "`max_epochs` was not set. Setting it to 1000 epochs.",
```